### ISSUE TITLE
Make it possible to consume different network (e.g. Ropsten testnet)

### ISSUE THREAD

**Issue by AndreMiras** [corpetty/py-etherscan-api#24]
It might be useful for developers to have access to the Etherscan Ropsten API.
Base URL is https://api-ropsten.etherscan.io/api
Documentation can be found here:
https://ropsten.etherscan.io/apis

**Comment by corpetty** [comment]
Yeah, they incorporated those API functionality since I've used this.  It would be good to include them.

**Comment by corpetty** [comment]
#41 is working on getting ropsten in there
